docker_files/httpd.conf

```diff
@@ -0,0 +1,98 @@
+ServerTokens OS
```

Comment:
I think there is a lot of stuff from production that needs to be added to this.
@klibby or @gozer can you look over this httpd.conf?